src/php/k8s-php-install.sh

```diff
@@ -0,0 +1,79 @@
+#!/bin/sh
+
+ZEND_API_NO=$1 # zend representation of the PHP version number
```

Comment:
How is this script being called and what mechanism is being used to supply the `ZEND_API_NO`?

Reply:
https://github.com/newrelic/k8s-agents-operator/pull/79/files#diff-9d8393bab101cb85655446396e995080f844370d607c6b39ae4e560f684e138aR82